app/views/public_pages/maintenance.html.erb

```diff
@@ -1,5 +1,7 @@
 <% content_for :page_title, t("views.ctc.questions.at_capacity.title") %>
-<% content_for :header, "&nbsp;" %>
```

Comment:
Silly question - since a nbsp is a non-breaking space, can we use a regular space? If so:

```
<% content_for :header, " " %>
```

If we don't care about any contents, can we make it empty, like:

```
<% content_for :header do %>
<% end %>
```

Or even:

```
<% content_for :header, "" %>
```

...in fact, why do we even bother making the header at all? Can we remove the line entirely?

What you have is OK, too.

Reply:
the reason we bother with the content_for header is that we are trying to hide the default header because it has branding for getctc or getyourrefund and this is supposed to be a generic maintenance page for both. (in application.html.erb, it does a check for content_for and otherwise renders the GYR header)

i tried all of those options and it turns out that none of them successfully replace the header. i expected it for the empty block because i saw something on the internet about that but i'm mildly surprised that the regular space (" ") didn't do it 🤷🏻‍♀️

Reply:
i realized after typing that that in the absence of a header there's no way to translate the page into spanish, so i talked to kelly and we're gonna use a ctc header instead

Reply:
```
<% content_for :header do %>
&nbsp;
<% end %>
```

Also OK!

Reply:
Space probably doesn't work because it's an ERB/Rails thing, and it's probably using `.present?`, and

```
irb(main):001:0> ' '.present?
=> false
```